app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -1,18 +1,18 @@
 class ManageIQ::Providers::Redhat::InfraManager < ManageIQ::Providers::InfraManager
-  require_nested :EventCatcher
-  require_nested :EventParser
-  require_nested :RefreshWorker
-  require_nested :RefreshParser
-  require_nested :MetricsCapture
-  require_nested :MetricsCollectorWorker
-  require_nested :Host
-  require_nested :Provision
-  require_nested :ProvisionViaIso
-  require_nested :ProvisionViaPxe
-  require_nested :ProvisionWorkflow
-  require_nested :Refresher
-  require_nested :Template
-  require_nested :Vm
+  require_nested  :EventParser
+  require_nested  :RefreshWorker
+  require_nested  :RefreshParser
+  require_nested  :MetricsCapture
+  require_nested  :MetricsCollectorWorker
+  require_nested  :Host
+  require_nested  :Provision
+  require_nested  :ProvisionViaIso
+  require_nested  :ProvisionViaPxe
+  require_nested  :ProvisionWorkflow
+  require_nested  :Refresher
+  require_nested  :Template
+  require_nested  :Vm
+  include_concern :ApiIntegration
```

Comment:
Is there a reason of moving this to a module? I find it harder to review this way, because I dont know what actually changed.